### PR TITLE
BusinessDashboard Theme | <Thumbnail> - Apply overrides to all size thumbnail

### PR DESCRIPTION
**Pull request by shelly-goldblit**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6992/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6992

File: packages/wix-style-react/src/Themes/businessDashboard/docs/components/Thumbnail.js [hunk 1]
```diff
@@ -2,13 +2,22 @@ export const playground = `
...
       </Cell>
       <Cell span={2}>
-        <Thumbnail size="small" disabled title="Disabled 3" />
+        <Thumbnail size="small" disabled title="Small Disabled 3" />
+      </Cell>
+      <Cell span={2}>
+        <Thumbnail title="Thumbnail 4" />
+      </Cell>
+      <Cell span={2}>
+        <Thumbnail selected title="Selecetd 5" />
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/docs/components/Thumbnail.js:17]: typo in selected

Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/docs/components/Thumbnail.js:17]: if the top are "Small" what are those? do you plan to support more sizes?

Comment (shelly-goldblit) [reply packages/wix-style-react/src/Themes/businessDashboard/docs/components/Thumbnail.js:17]: there are default sizes

Comment (shelly-goldblit) [reply packages/wix-style-react/src/Themes/businessDashboard/docs/components/Thumbnail.js]: All sizes are supported

Comment (adicicurel) [reply packages/wix-style-react/src/Themes/businessDashboard/docs/components/Thumbnail.js]: Can you add all of the supported sized to the playground?

Comment (shelly-goldblit) [reply packages/wix-style-react/src/Themes/businessDashboard/docs/components/Thumbnail.js]: Already done

**Review by adicicurel — APPROVED** [review]

**Review by adicicurel — APPROVED** [review]

**Pull request merged**